README.md

```diff
@@ -200,13 +202,14 @@ Many C++20 features could be considered truly revolutionary, changing old good C
 * [Exploring C++20: The Programmer's Introduction to C++](https://www.amazon.com/dp/1484259602) (Ray Lischner, 2020)
 * [The C++ Standard Library](https://leanpub.com/cpplibrary) (Rainer Grimm, 2021)
 * [Programming with C++20](https://leanpub.com/programming-with-cpp20) (Andreas Fertig, 2021)
-* [C++20 - The Complete Guide](https://leanpub.com/cpp20) (Nicolai M. Josuttis, the book is 90% complete as of May 2022)
+* [C++20 - The Complete Guide](https://leanpub.com/cpp20) (Nicolai M. Josuttis)
 * [C++ Concepts](https://leanpub.com/cppconcepts) (Sandor Dargo, 2022)
 
 ## Software Design and Design Patterns in C++
 
 Software design and architecture in C++ came a long way from the GoF book, released in 1994. Since then, some design patterns became just language features, some are changed beyond recognition - that's why it's vital to consider new language features, thinking of implementing a particular software design solution.
 
+* [Hands-On Design Patterns with C++ 2nd Edition](https://www.amazon.com/dp/1804611557) (Fedor G. Pikus, 2023)
```

Comment:
I find this book the best quality book in C++ design patterns, way much better than Dmitri's book below.
It also covers latest features up to C++20.